src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java

```diff
@@ -89,9 +113,12 @@ private ReadFilterLibrary(){ /*no instance*/ }
                     (! read.isUnmapped() && read.getContig().equals(read.getMateContig()));}}
 
     /**
-     * Reads that have a mapped mate and both mate and read are on different same strands (ie the usual situation).
+     * Keep paired reads (0x1) if it is mapped, its mate is mapped (read is not 0x8) and both the read and
```

Comment:
"For paired reads, keep only reads that are mapped, have a mate that is mapped, and both ..."